test/formatters/junitFormatterTests.ts

```diff
@@ -106,14 +106,24 @@ describe("JUnit Formatter", () => {
                 </testsuite>
             </testsuites>`.replace(/>\s+/g, ">"); // Remove whitespace between tags;
 
-        assert.equal(formatter.format(failures), expectedResult);
+        assert.equal(formatter.format(failures, [], [TEST_FILE_1, TEST_FILE_2]), expectedResult);
     });
```

Comment:
Could you please add a unit test for a result where 1 file has a failure and 2 file does not? Just to make sure both get into the output.

Reply:
Done :)